package.json

```diff
@@ -7,9 +7,11 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc -t node_modules/docdash --readme JSDOC.md -c jsdoc-config.json",
     "doclint": "npm run doc -- -t templates/silent",
-    "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-unit",
+    "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-with-coverage",
     "test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
     "test-examples": "mocha -t 300000 --recursive test/examples",
+    "test-with-coverage": "nyc -n src -r html cross-env BABEL_DISABLE_CACHE=1 mocha --require babel-core/register test/*unit_test.js",
```

Comment:
Could this just call `npm run test-unit`? (I suppose not, otherwise you'd done it; just checking that it's not an oversight)